index.html

```diff
@@ -6,10 +6,176 @@
+                    <a href=""><img class="speaker" src="./image/speaker_04.png" alt="Speaker 1 photo"></a>
+                    <div class="speaker-info p-2">
+                        <h3 class="text-dark">Julia Leda</h3>
+                        <p class="speaker-prof fst-italic">president of young pirates of Europe.</p>
+                        <p>European ingetration, political democracy and participation of youth through online as her major concern. Reda's report outlining potential changes to EU copyright law was approved by the parliament by July.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="speaker-5">
+                <div class="d-flex justify-content-center justify-content-between p-4">
+                    <a href=""><img class="speaker" src="./image/speaker_05.png" alt="Speaker 1 photo"></a>
+                    <div class="speaker-info p-2">
+                        <h3 class="text-dark">Lila Tretikov</h3>
+                        <p class="speaker-prof fst-italic">Executive Director of Wikimedia Foundation</p>
+                        <p>Lila Tretikov Executive Director of Wikimedia Foundation, non profit organization that operats wikipedia. Wekipedia is freely availble in 290 languages and used by half billion people around the world every month.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="speaker-6">
+                <div class="d-flex justify-content-center justify-content-between p-4" >
+                    <a href=""><img class="speaker" src="./image/speaker_06.png" alt="Speaker 1 photo"></a>
+                    <div class="speaker-info p-2">
+                        <h3 class="text-dark">Ryan Merkley</h3>
+                        <p class="speaker-prof fst-italic">CEO of Creative Commons. Ex COO of Mozila foundation</p>
+                        <p>Ryan had been leading open source project at the Mozila foundation such as the open sourse moment.</p>
+                    </div>
+                </div> 
+            </div>
+        </div>
+        <div class="dropdown-arrow-div">
+            <div class="dropdown-arrow d-flex justify-content-center border p-4">
+                <img src="./image/down_arrow.png" alt="">
+            </div>
+        </div>                  
+    </div>
+    <div class="partner" id="partner">
+        <h2 class="partners_heading text-center text-white pt-4">Partners</h2>
+        <h3 class="text-danger text-center">____</h3>
+        <div class="parner-logo d-flex justify-content-center justify-content-around flex-wrap p-4 gap-2">
+            <h2 class="text-light">mozilla</h2>
+            <h2 class="fs-2 font-monospace fw-light">Google</h2>
+            <h2 class="fw-bolder fs-1">NAVER</h2>
+            <h2 class="fs-3">daumkkao</h2>
+            <h2 class="">airbnb</h2>
+        </div>
+    </div>
```

Comment:
- [ ] Your partners section seems to disappear for the home page in the desktop version as you can see below. Kindly fix this small visual bug

![image](https://user-images.githubusercontent.com/90556221/183044895-77043360-8a7b-4e42-9604-ebf348a3e351.png)

Reply:
Adjusted the same.